core/templates/pages/skill-editor-page/editor-tab/skill-editor-main-tab.directive.spec.ts

```diff
@@ -92,6 +92,16 @@ describe('Skill editor main tab directive', function() {
     expect($scope.hasLoadedSkill()).toBe(true);
   });
 
+  it('should update publish button when data changes on prerequisite skills',
+    fakeAsync(() => {
+      let spyApply = spyOn($scope, '$apply');
+
+      $scope.updateViewPublishButton();
```

Comment:
can you point me to where this function exists? I am not able to find, thanks!

Reply:
line no. 86 `skill-editor-main-tab.directive.ts` file

Reply:
@aishwary023 PTAL

Reply:
Can you send a permalink for the same?

Reply:
https://github.com/oppia/oppia/pull/14505/files#diff-76b94f3fff8267442a4e902e45d63aca0368fb7cc76567138a042c8efd677be8
@aishwary023  PTAL